spec/features/admin/merchants/show_spec.rb

```diff
@@ -2,18 +2,32 @@
 
 RSpec.describe "Admin Merchants Show" do
   before(:each) do
-    @merchants = create_list(:merchant, 3)
-    @merchant1 = @merchants[0]
-    @merchant2 = @merchants[1]
-    @merchant3 = @merchants[2]
+    @merchant1 = Merchant.create(name: "Amazon")
+    @merchant2 = Merchant.create(name: "Walmart")
+    @merchant3 = Merchant.create(name: "Target")
   end
 
-  describe '#us 25' do
-    it 'see a specific merchants show page ' do
-      # When I click on the name of a merchant from the admin merchants index page (/admin/merchants),
-      visit admin_merchants_path(@merchant1)
-      # Then I am taken to that merchant's admin show page (/admin/merchants/:merchant_id)
-      # And I see the name of that merchant
+  describe '#us 26' do
+    it 'Update the merchant' do
+
+      # When I visit a merchant's admin show page (/admin/merchants/:merchant_id)
```

Comment:
I don't mind the commented user stories during testing process. However, I think we should take them out before we turn in the project so we don't have a whole bunch of extra lines of code. tests look great other than that though.